README.md

```diff
@@ -1,16 +1,20 @@
 # `nixfmt`
 
-`nixfmt` is a formatter for Nix code, intended to easily apply a uniform style.
-
-You are encouraged to test this out on your code and submit any undesirable formatting you find as an issue
+`nixfmt` is the official formatter for Nix language code, intended to easily apply a uniform style.
 
 ![Build Status](https://github.com/NixOS/nixfmt/actions/workflows/main.yml/badge.svg?branch=master)
 
 ## State
 
-`nixfmt` will form the basis for the initial official standard Nix formatter, as established by [RFC 166](https://github.com/NixOS/rfcs/pull/166).
+`nixfmt` was originally developed by [Serokell](https://github.com/serokell).
+It was used as the basis for the official standardised Nix formatter, as established by [RFC 166](https://github.com/NixOS/rfcs/pull/166).
+
+The official standard differs considerably from the original implementation.
+Be aware of this if you track the [`master`](https://github.com/NixOS/nixfmt/tree/master) branch.
+Until the [next release](https://github.com/NixOS/nixfmt/issues/272), `nixfmt` should be considered **very unstable**.
```

Comment:
Slight wording tweak:
```suggestion
Until a new [stable release](https://github.com/NixOS/nixfmt/issues/272) is made, `nixfmt` should be considered **very unstable**.
```

Reply:
Yeah but "new" can also be read as "latest", i.e. already there, while "next" is clearly about the future. This should be possible to infer without reading the linked issue. Also in this context there's no way to know what makes a release "stable".

Reply:
```suggestion
Until the [next release](https://github.com/NixOS/nixfmt/issues/272), expect `nixfmt` to be subject to change.
```

How about this?